util/Migrator/DbScripts/2025-07-18_00_AddReadManyOrganizationIntegrationAndConfigurationByIdProcs.sql

```diff
@@ -0,0 +1,29 @@
+CREATE PROCEDURE [dbo].[OrganizationIntegration_ReadByOrganizationId]
```

Comment:
```suggestion
CREATE OR ALTER PROCEDURE [dbo].[OrganizationIntegration_ReadByOrganizationId]
```
Needs to be backward-compatible. Do this below too.